### ISSUE TITLE
npm install after clone failing

### ISSUE THREAD

**Issue by njappboy** [LiskHQ/lisk#739]
On ubuntu xenial 64 I receive this error after cloning the lisk repository and trying to install npm dependencies. All prerequisites have been installed and all operations are occurring inside a virtualized vagrant environment via virtualbox.

```
npm i

# removed for brevity
npm ERR! git rev-list -n1 optional_data_test: fatal: ambiguous argument 'optional_data_test': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 optional_data_test: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 optional_data_test: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 optional_data_test:
npm ERR! Linux 4.4.0-71-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "i"
npm ERR! node v6.11.2
npm ERR! npm  v3.10.10
npm ERR! code 128

npm ERR! Command failed: git rev-list -n1 optional_data_test
npm ERR! fatal: ambiguous argument 'optional_data_test': unknown revision or path not in the working tree.
```

```
>npm -v
3.10.10

>nodejs -v
v6.11.2
```

**Comment by Isabello** [comment]
What branch?

**Comment by willclarktech** [comment]
Presumably `development`. This will be fixed in `development` branch once https://github.com/LiskHQ/lisk/pull/740 is merged. In the meantime, @njappboy what's your use case? `development` is almost certainly not the branch you want.

**Comment by willclarktech** [comment]
`optional_data_test` branch has been restored in `lisk-js` so this issue should be resolved while we're waiting for #740 anyway.

**Comment by karmacoma** [comment]
Assuming issue resolved. Therefore closing.
